packages/backend/src/main.ts

```diff
@@ -2,13 +2,21 @@ import { NestFactory } from '@nestjs/core';
 import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 require('dotenv').config();
-import { Logger } from '@nestjs/common';
+import { Logger,ValidationPipe } from '@nestjs/common';
+import * as helmet from 'helmet';
+import * as csurf from 'csurf';

```

Comment:
nieużywany import, do usunięcia

Reply:
z racji tego, że  csurf powinno być użyte to import dobrze że jest

Reply:
ok